shared/wallets/asset/index.js

```diff
@@ -154,3 +152,63 @@ const styles = styleSheetCreate({
 })
 
 export default Asset
+
+type AssetWrappedState = {
+  expanded: boolean,
+}
+
+export class AssetWrapped extends React.Component<Props, AssetWrappedState> {
```

Comment:
seems like we should just merge this into Asset and not have this composition

Reply:
@chrisnojima Ah, I did this because the storybook and flowtype expect an "expanded" prop to be passed in to Asset that controls whether it's expanded.  If it's just local state inside Asset, should it still accept the prop?  If not, how would we get storybook to show both expanded and non-expanded separately?  Maybe there should be a prop, but it only controls the initial value, and after that it just uses state..?

Reply:
maybe best if its always a prop and expanded is controlled by the parent. so the list has a map of the expanded ids and it passes it down and there's a prop it passes down to toggle

Reply:
@chrisnojima Hm, I've made all of the suggested changes except this one about where the `expanded` prop comes from.  Do you think it's important to change?  One reason it's awkward is that assets don't actually have IDs coming through from the service to store in our map (perhaps they should?), it's purely an array.

Reply:
k we can leave it